### ISSUE TITLE
Project document issues

### ISSUE THREAD

**Issue by suziyou12** [minhtriet/gas_market#1]
Step 5 of the installation process requires a configuration file, but there is no configuration file in the project.

**Comment by minhtriet** [comment]
You are absolutely right. Could you then run the project, add requirements and do a `pip freeze`? I would very much welcome a PR.

You would need pandas, keras, spacy, nltk, and some other minors library

**Comment by AdamLeopold** [comment]
Could you please inform what exactly is the requirements

**Comment by minhtriet** [comment]
what you need may include 
```
spacy
keras
matplotlib
nltk
pandas
```

Let me know if you need anything else
